src/lib/tableland-calls.ts

```diff
@@ -120,6 +121,19 @@ async function query(
   return json as ReadQueryResult;
 }
 
+async function write(
+  this: Connection,
+  query: string
+): Promise<WriteQueryResult | null> {
+  const message = await GeneralizedRPC.call(this, "relayWriteQuery", {
+    statement: query,
+  });
+  const response = await SendCall.call(this, message);
+  const json = await sendResponse(response);
+
+  return json as WriteQueryResult;
```

Comment:
Here, I'd just `return` and have the response be a `Promise<void>`. Line 132 could also just be `await sendResponse(response);`

Reply:
If the returned value of a write _does_ provide useful information about the right (rows written or something?), then we _should_ return this, because it provides useful information. Otherwise, don't.

Reply:
The validator RPC response is `{data: null}` which does seem like we might as well return void.  
I recognize I'm throwing a jellyfish here, but it seems related and worth mentioning.  Now that the validator is acting as a relay to the Smart Contract and not actually doing the write until the SC emits the event, this function is returning before we have full confidence the write is finished.

Reply:
Ah gotcha, ok. Let's just document it for now then (on the method itself as doc strings), and then you can create a tension for it when you have time 👍 .

Reply:
Example: Note: This method returns right away, once the write request has been sent to a validator for writing to the Tableland smart contract. However, the write is not confirmed until a validator has picked up the write event from the smart contract, and digested the event locally.

Reply:
Sorry this thread is getting long...
I was in process of making these updates, and I see that the validator now returns the transaction hash.  I feel like it would make sense for the SDK to return that as either a string or an object with a single property. Thoughts?